qrcode/image/base.py

```diff
@@ -36,17 +38,19 @@ def drawrect(self, row, col):
         Draw a single rectangle of the QR code.
         """
 
-    def drawrect_context(self, row: int, col: int, qr: "QRCode"):
+    def drawrect_context(self, row: int, col: int, qr: QRCode):
         """
         Draw a single rectangle of the QR code given the surrounding context
         """
-        raise NotImplementedError("BaseImage.drawrect_context")  # pragma: no cover
+        msg = "BaseImage.drawrect_context"
+        raise NotImplementedError(msg)  # pragma: no cover
 
     def process(self):
         """
         Processes QR code after completion
         """
-        raise NotImplementedError("BaseImage.drawimage")  # pragma: no cover
+        msg = "BaseImage.drawimage"
+        raise NotImplementedError(msg)  # pragma: no cover
```

Comment:
Ties into the other comment about the split-exception-message-from-the-raise rule above; you'd now need to add `# pragma: no cover` to the `msg` lines too, since they'll never be executed either...